### ISSUE TITLE
Invalidate 8-digit Brazil mobile numbers (BR, +55)

### ISSUE THREAD

**Issue by keghani** [googlei18n/libphonenumber#1618]
These numbers have been officially documented as out of use for a while.
We're getting reports that carriers are no longer supporting them either, so the time is ripe to stop validating them.

**Comment by jantonho** [comment]
Hello @keghani, you have planning to implement this issue?

**Comment by keghani** [comment]
Yes. Since I filed this we've invalidated 2x, 9x, and 8x 8-digit mobile subscriber numbers. 1x 8-digit mobile subscriber numbers were already invalid.

As of now 3x, 7x, 6x, 4x, and 5x remain valid in the pre-transition form. We will invalidate them in stages to minimize risk. Not all ranges are at the same stage, the post-transition numbers were assigned in this order with the entire process spanning several years.

We'll update this issue when all the pre-transition numbers are invalid in the library.

**Comment by anu-branco** [comment]
Hi, I am using the release v8.8.1 (Sep 05, 2017) and Brazilian 9x 8digit mobile subscriber numbers are still valid for some area codes. 
Ex: 34 9100 1234 or 61 9100 1234. Where 34 and 61 are national area codes.
But for some area code like (11) and (12) they are not (what is correct).

As regulated by ANATEL (the national agency for telecommunications) all mobile subscriber numbers are 9 digits now and the carriers are no longer supporting 8 digit mobile number.

**Comment by keghani** [comment]
Hi @anu-branco,

Re. "what is correct" please see my above response.
We have not made any updates since then, and the status is the same. We're working on it. We'll update this issue when all pre-transition numbers are invalid in the library.

**Comment by anu-branco** [comment]
Thanks for your prompt answer @keghani ,

  What I wanted to point out is that the [transition period is over for the whole country(Brazil)]((http://www.anatel.gov.br/Portal/exibirPortalNivelDois.do?codItemCanal=1794&nomeVisao=Cidad%E3o&nomeCanal=Telefonia%20M%F3vel&nomeItemCanal=Nono%20D%EDgito)).
  So 8 digit mobile subscriber numbers are all history now.

**Comment by keghani** [comment]
Thanks, I understand that. This issue is tracking invalidation of all the 8-digit numbers under mobile (except mobile radio services, where the subscriber number starts with 7). So we *are* going to fix this, and we know we should, we're still working on it though.
